src/classes/ADDR_Contact_TEST.cls

```diff
@@ -0,0 +1,1112 @@
+            contact.MailingStreet = 'new street\r\nsecond line';
+            contact.MailingCity = 'new city';
+        }
+        Test.startTest();
+        insert contacts; 
+        Test.stopTest();       
+        
+        // verify the Address objects split street.
+        List<Address__c> ListAddr = [select Id, MailingStreet__c, MailingStreet2__c, Formula_MailingStreetAddress__c from Address__c];
+        System.assertEquals(2, ListAddr.size());
+        for (Address__c addr : ListAddr) {
+            system.assertEquals('new street', addr.MailingStreet__c);
+            system.assertEquals('second line', addr.MailingStreet2__c);
+            system.assertEquals('new street, second line', addr.Formula_MailingStreetAddress__c);
+        }
+
+        map<Id, Contact> mapAccIdAcc = new map<Id, Contact>([select Id, Name, MailingStreet, MailingCity, MailingState, 
+        MailingPostalCode, MailingCountry, MailingLatitude, MailingLongitude, is_Address_Override__c, Current_Address__c from Contact]);
+        
+        for (Contact contact : mapAccIdAcc.values()) {
+            system.assertEquals('new street\r\nsecond line', contact.MailingStreet);
+            system.assertEquals(false, contact.is_Address_Override__c);
+            system.assertNotEquals(null, contact.Current_Address__c);
+        }
+    }
+
+    /*********************************************************************************************************
+    @description Insert new default multiline street addresses w/ existing default addresses  
+    verify:
+        Contact address matches new default
+        street & street2 combined into contact and Contact single street field
+    **********************************************************************************************************/            
+    @isTest
+    public static void insertNewDefaultMultilineStreetAddr() {
+        configSettings();
+                
+        // this creates a default Address for each Contact
+        UTIL_UnitTestData_TEST.ContactsWithAddrs contactsAddrs = UTIL_UnitTestData_TEST.createTestContactsAddrs(2);
+
+        // create additional addresses
+        List<Address__c> addrs = UTIL_UnitTestData_TEST.initTestAddr(2);
+        for (Integer i = 0; i < contactsAddrs.contacts.size(); i++) {
+            addrs[i].Parent_Contact__c = contactsAddrs.contacts[i].Id;
+            addrs[i].Default_Address__c = true;
+            addrs[i].MailingStreet__c = 'New Default Street';
+            addrs[i].MailingStreet2__c = 'Second Line';
+            addrs[i].MailingCity__c = 'New Default City' + i;
+        }
+        ADDR_Addresses_TDTM.hasRunAddrTrigger = false;
+        Test.startTest();
+        insert addrs;
+        ADDR_Addresses_TDTM.hasRunAddrTrigger = false;        
+        Test.stopTest();
+
+        // verify Contact address is new
+        map<Id, Contact> mapAccIdAcc = new map<Id, Contact>([select Id, Name, MailingStreet, MailingCity, MailingState, 
```

Comment:
If the table in this query contains more than 50'000 records a runtime exception will be thrown. I suggest to introduce a `WHERE` clause or cap the max number of records returned adding a `LIMIT` clause to the query.